ansible/roles/docker_py/tasks/main.yml

```diff
@@ -1,11 +1,10 @@
 ---
 
-- name: installs apt-https-transport and required kernel packages
+- name: installs apt-https-transport
   apt: >
     name={{ item }}
     state=present
   with_items:
-    - "linux-image-{{ ansible_kernel }}"
```

Comment:
I can confirm that they changed their base image in a way that this is no longer a requirement.